conf/messages

```diff
@@ -88,14 +88,20 @@ preRequisites.countryOfDispatch.placeholder = Select a country
 preRequisites.countryOfDispatch.error.required = Enter the country of dispatch
 preRequisites.countryOfDispatch.paragraph = This is the country the item was originally dispatched from. It may not be where the transit movement begins.
 
-preRequisites.itemsDestinationCountry.title = What is the country of destination?
-preRequisites.itemsDestinationCountry.heading = What is the country of destination?
+preRequisites.addCountryOfDestination.title = Are the goods being transported to another country after the end of this transit movement?
+preRequisites.addCountryOfDestination.heading = Are the goods being transported to another country after the end of this transit movement?
+preRequisites.addCountryOfDestination.paragraph = This means after the goods have arrived at the office of destination and have been processed by Customs.
+preRequisites.addCountryOfDestination.checkYourAnswersLabel = Are the goods being transported to another country after the end of this transit movement?
+preRequisites.addCountryOfDestination.change.hidden = if the goods are being transported to another country after the end of this transit movement
+preRequisites.addCountryOfDestination.error.required = Select yes if the goods are being transported to another country after the end of this transit movement
+
+preRequisites.itemsDestinationCountry.title = What country are the items being transported to?
+preRequisites.itemsDestinationCountry.heading = What country are the items being transported to?
 preRequisites.itemsDestinationCountry.hint = Enter the country, like France or Portugal.
-preRequisites.itemsDestinationCountry.paragraph = This is where the goods complete their journey. It may not be where the transit movement ends.
-preRequisites.itemsDestinationCountry.checkYourAnswersLabel = Country of destination
-preRequisites.itemsDestinationCountry.change.hidden = country of destination
+preRequisites.itemsDestinationCountry.checkYourAnswersLabel = Country items are being transported to
```

Comment:
Maybe worth asking Gavin if a "the" is missing here i.e. `Country the items are being transported to` to be consistent with the other message keys